docs/man_pages/project/creation/init.md

```diff
@@ -0,0 +1,20 @@
+init
+==========
+
+Usage | Synopsis
+---|---
+General | `$ tns init [--path <Directory>] [--force]`
+
+Initializes a project for development. This will ask you a bunch of questions, and then write a package.json for you.
```

Comment:
"ask you a bunch of questions" does not look very professional to me. Perhaps "it will ask you interactively to configure the project". Ping @ikoevska